scss/_patterns_side-navigation.scss

```diff
@@ -525,6 +525,10 @@
     padding-left: $spv--small;
   }
 
+  .p-side-navigation__heading--linked {
+    max-width: none;
```

Comment:
SCSS linting is failing on this one, because we already have `.p-side-navigation__heading--linked` defined below, so these could be joined together.

But to make it even better, I think we can put the max-width inside the `%side-navigation__heading` definition earlier in the file, so it will apply to ALL headings (not just linked).